.github/workflows/release.yaml

```diff
@@ -44,6 +44,10 @@ jobs:
           images: ghcr.io/adfinis/timed-backend
           flavor: |
             latest=auto
+          tags: |
+            type=semver,pattern={{version}},value=${{ steps.semrel.outputs.version }}
+            type=semver,pattern={{major}}.{{minor}},value=${{ steps.semrel.outputs.version }}
+            type=semver,pattern={{major}},value=${{ steps.semrel.outputs.version }}
```

Comment:
don't we also need {{major}}.{{minor}}.{{patch}}?

Reply:
thats what `version` is for AFAIK

Reply:
![image](https://github.com/adfinis/timed-backend/assets/110528300/e2df9ea4-67da-4a51-a639-4e18fd331af0)
https://github.com/docker/metadata-action?tab=readme-ov-file#typesemver

Reply:
tru dat